packages/nx-plugin/src/generators/app/files/template-angular-v16/src/app.config.server.ts__template__

```diff
@@ -0,0 +1,10 @@
+import { ApplicationConfig, mergeApplicationConfig } from '@angular/core';
+import { provideServerRendering } from '@angular/platform-server';
+import { provideClientHydration } from '@angular/platform-browser';
+import { appConfig } from './app.config';
+
+const serverConfig: ApplicationConfig = {
+  providers: [provideServerRendering(), provideClientHydration()],
```

Comment:
provideClientHydration() is already included in the app.config.ts. It doesn't need to be included here

```suggestion
  providers: [provideServerRendering()],
```

Reply:
fixed